fixtures/net/ip_vs

```diff
@@ -12,3 +12,7 @@ TCP  C0A80039:0CEA wlc
 TCP  C0A80037:0CEA wlc  
   -> C0A8321A:0CEA      Tunnel  0      0          0         
   -> C0A83120:0CEA      Tunnel  100    0          0         
+TCP  [2620:0000:0000:0000:0000:0000:0000:0001]:0050 sh 
```

Comment:
Hm, mixing different formats in the same file is confusing, as it wouldn't happen in real-life. I'd prefer different fixture files instead. The current fixtures structure needs some clean up though, so I'd be fine postponing that.